akka-actor/src/main/resources/reference.conf

```diff
@@ -705,13 +705,29 @@ akka {
     #
     # Please note that this option does not stop you from manually invoking java serialization
     #
-    # The default value for this might be changed to off in future versions of Akka.
-    allow-java-serialization = on
+    allow-java-serialization = off
+
+    # Log warnings when the default Java serialization is used to serialize messages.
```

Comment:
Do we even want to use the phrase `default Java serialization` vs drop the default?

Reply:
yes, should be dropped.
fixed